index.html

```diff
@@ -0,0 +1,368 @@
+
+<img src="images/horizontal-fold.svg" height="150px" alt="">
+<img src="images/vertical-fold.svg" height="150px" alt="">
+
+*<br>An illustration of two [=foldable-screen device=] concepts in two different [=postures=]. ([Source](https://w3c.github.io/screen-fold/))*
+
+<aside class="note">
+Traditional single-screen laptop form factor is excluded from the foldable device definition and web specifications that refer to this form factor specifically refer to it as a <dfn>traditional laptop</dfn>. Wikipedia has a more nuanced definition of laptop types, but for the purpose of [related web specifications](#related-web-specifications) [all laptop types](https://en.wikipedia.org/wiki/Laptop#Types) are considered traditional laptops.
+</aside>
+
+### Dual-screen device
+
+A <dfn data-export="">dual-screen device</dfn> has two or more physical internal screens made of solid materials placed together to form a larger foldable virtual screen. A defining characteristic of this form factor is a physical hinge that can rotate from 0 degree to at most 360 degrees that forms a physical gap in between the screens. The range of the hinge movement as well as the width of the gap varies depending on the hardware design. The screens separated by the physical hinge are made of solid materials, typically glass, and as such are not flexible. This design requires web developers to work around the fact there is a physical gap between the screens on both sides of the fold.
+
+![](images/dual-screen-device.png)
+*<br>A [=dual-screen device=] folded open. ([Source](TODO))*
+
+### Foldable-screen device
+
+A <dfn data-export="">foldable-screen device</dfn> has one or multiple screens made of foldable material that can bend so that the screen can fold onto itself. The hinge can typically rotate from 0 degree to 180 degrees depending on the flexible screen materials’ capability of sustaining the bend radius. The material around the area of the screen that can bend is often softer than on the other parts of the screen. To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions. This makes it easier for existing web content to be presented without visual distractions when the foldable-screen device is folded open into so-called flat posture.
```

Comment:
- statement 1:
 "A foldable-screen device has one or multiple screens made of foldable material that can bend so that the screen can fold onto itself."

- Statement 2:
"To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions."

How does it work to have a multiscreen with no visible hinge?

Reply:
Are you concerned about the "or multiple" bit in 1? I clarified that by "the each screen can". There could be a device with two or more foldable screens in one device.

Re 2, can you specify the unclear bit?

Reply:
sorry for the confusion.
In this foldable-screen device description, there are, at the moment 2 statements defining how they should look like to be qualified as foldable-screen device:
1) "A foldable-screen device has one or multiple screens made of foldable material that can bend so that the screen can fold onto itself."
That s fine and that s also valid as a generic statement for all generic foldable devices.

but 2) "To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions."

That s also fine when we are talking about a single "flexible" screen, having a flex material in the middle such as "Samsung galaxy fold".

But what kind of device would be a foldable-screen device, that has more than 1 screen, with no physical gap? (complying with both statements above)? Do you have an existing example?

Reply:
Thanks for paying attention to the details. Addressed by 9dbff3e302507ed7a7b1ce2b124cd61303622b83.